### PR TITLE
SOLR-10288 remove non minified js

### PR DESCRIPTION
**Pull request by MarcusSorealheis**
# Description

removing full files from index.html.

# Solution

add the full suffix to the non-minified files so that we can programmatically exclude them in the build file.

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

* [x] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
* [x] I have created a Jira issue and added the issue ID to my pull request title.
* [x] I am authorized to contribute this code to the ASF and have removed any code I do not have a license to distribute.
* [x] I have developed this patch against the `master` branch.
* [ ] I have run `ant precommit` and the appropriate test suite.
* [ ] I have added tests for my changes.
* [ ] I have added documentation for the Ref Guide (for Solr changes only).

### CODE REVIEW

**Comment by MarcusSorealheis** [comment]
I still need to figure out how to properly exclude the full files in the build.xml before this is merged. I need help. I don't `ant` at all. @erikhatcher

**Comment by erikhatcher** [comment]
Let's just remove the non-minimized ones (and correct any references to use the minimized).  Any reason to keep the non-minimized here at all?

**Comment by erikhatcher** [comment]
Maybe something like this? 

```$ git diff
diff --git a/solr/webapp/build.xml b/solr/webapp/build.xml
index 99f8d17b421..0d60d8f3740 100644
--- a/solr/webapp/build.xml
+++ b/solr/webapp/build.xml
@@ -43,7 +43,7 @@
     <ant dir="${common-solr.dir}" inheritall="false" target="contribs-add-to-webapp"/>
     <mkdir dir="${server.dir}/solr-webapp/webapp"/>
     <copy todir="${server.dir}/solr-webapp/webapp">
-      <fileset dir="web" excludes="${exclude.from.webapp}"/>
+      <fileset dir="web" excludes="${exclude.from.webapp},libs/angular.js,libs/angular-cookies.js,libs/angular-route.js,libs/angular-sanitize.js,libs/angular-utf8-base64.js,libs/chosen.jquery.js"/>
       <fileset dir="${dest}/web" excludes="${exclude.from.war}"/> <!-- contribs' additions -->
     </copy>
     <mkdir dir="${server.dir}/solr-webapp/webapp/WEB-INF/lib"/>
(base) Eriks-MBP:solr erikhatcher$ ls server/solr-webapp/webapp/libs/
angular-chosen.js		angular-sanitize.min.js		d3.js				jquery-ui.min.js
angular-cookies.min.js		angular-utf8-base64.min.js	highlight.js			jquery.jstree.js
angular-resource.min.js		angular.min.js			jquery-1.7.2.min.js		ngtimeago.js
angular-route.min.js		chosen.jquery.min.js		jquery-2.1.3.min.js```

File: solr/webapp/build.xml [hunk 1]
```diff
@@ -19,12 +19,12 @@
   <description>Solr webapp</description>
 
   <property name="rat.additional-includes" value="**"/>
-  <property name="rat.additional-excludes" value="web/img/**"/>
+  <property name="rat.additional-excludes" value="web/img/**,*full*"/>
```
Comment (erikhatcher) [inline solr/webapp/build.xml]: That's a red herring - or a RAT herring.   RAT is a license checker.

File: solr/webapp/web/index.html [hunk 2]
```diff
@@ -53,8 +53,8 @@
   <script src="libs/jquery-2.1.3.min.js"></script>
   <script src="libs/chosen.jquery.js"></script>
   <script src="libs/jquery.jstree.js"></script>
-  <script src="libs/angular.js"></script>
-  <script src="libs/angular-chosen.js"></script>
+  <script src="libs/angular.min.js"></script>
```
Comment (erikhatcher) [inline solr/webapp/web/index.html]: my diff in the comment didn't include this, but yeah this is of course necessary too.   there's some other references that would need updating too?

File: solr/webapp/build.xml (hunk 1 above)
Comment (MarcusSorealheis) [reply solr/webapp/build.xml]: oh got. my initial exclude commit broke my web app locally. was trying to figure out what would work nothing had yet.

**Comment by MarcusSorealheis** [comment]
@erikhatcher

**Comment by MarcusSorealheis** [comment]
Screen Shot 2019-07-13 at 11 16 43 AM

File: solr/webapp/web/index.html (hunk 2 above)
Comment (MarcusSorealheis) [reply solr/webapp/web/index.html]: no other references

**Pull request closed without merging**